src/components/Shared/SelectCollectModule/FeeEntry.tsx

```diff
@@ -71,7 +75,7 @@ const FeeEntry: FC<Props> = ({
         <ArrowLeftIcon className="w-4 h-4" />
         <div>Back</div>
       </button>
-      <Form form={form} className="space-y-4" onSubmit={({}) => {}}>
+      <Form form={form} className="space-y-4" onSubmit={handleSubmit}>
```

Comment:
**question:** Can't we add `//do nothing` here instead of creating a function? 🤔

Reply:
Yes, we can. Will make the change